react/src/components/molecules/ListingTable/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+
+import Collapse from '../../animations/Collapse';
+
+import './style.css';
+
+class ListingTableItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({
+      open: !this.state.open
+    });
+  }
+
+  render() {
+    const row = this.props.row;
+    const visibleItems = row.visibleItems || 2;
```

Comment:
can simplify this to `const { visibleItems } = row.visibleItems || 2;`

Reply:
This doesn't work because sometimes row.visibleItems is not set, so it can't be pulled out in this manner.